src/index.js

```diff
@@ -18,6 +18,7 @@ async function main() {
       setPagesPath({ staticSiteGenerator, generatorConfigFile, path: siteUrl.pathname })
     }
     outputPagesBaseUrl(siteUrl)
+    core.exportVariable('GITHUB_PAGES', true)
```

Comment:
Since environment variables are all strings, let's be a little more explicit with the data type here. 👍🏻 

```suggestion
    core.exportVariable('GITHUB_PAGES', 'true')
```

Reply:
Isn't that like saying to use a string literal instead of an enum?